app/controllers/generic_object_definition_controller.rb

```diff
@@ -23,6 +24,30 @@ def self.model
     GenericObjectDefinition
   end
 
+  def button
+    javascript_redirect(
+      case params[:pressed]
+        when 'generic_object_definition_new'
+          { :action => 'new' }
+        when 'generic_object_definition_edit'
+          { :action => 'edit', :id => from_cid(params[:id] || params[:miq_grid_checks]) }
+      end
+    )
+  end
+
+  def new
+    assert_privileges('generic_object_definition_new')
+    drop_breadcrumb(:name => _("Add a new Generic Object Class"), :url => "/generic_object_definition/new")
+    @in_a_form = true
+  end
+
+  def edit
+    assert_privileges('generic_object_definition_edit')
+    drop_breadcrumb(:name => _("Edit Generic Object Class"), :url => "/generic_object_definition/edit")
```

Comment:
Should this be `.../edit/#{params[:id]}`

Reply:
I think so.
Although the `:url` parameter here makes no difference, since the Edit page is the tail end of the breadcrumb trail.
Which means, we could also get rid of `:url` ...
But I'll fix it anyway and stay consistent with the other `drop_breadcrumb` calls.

Reply:
Fixed in 8e2993b